app/components/Vault/index.js

```diff
@@ -763,34 +759,46 @@ const Vault = (props) => {
     vaultAssets = truncateUsd(0);
   }
 
-  // FIXME: migration vaults
-  const retired = [
-    '0x7Ff566E1d69DEfF32a7b244aE7276b9f90e9D0f6', // crvSBTC v1
-    '0x5334e150B938dd2b6bd040D9c4a03Cff0cED3765', // crvRENBTC v1
-    '0xBacB69571323575C6a5A3b4F9EEde1DC7D31FBc1', // crvSAAVE v1
-    '0x7F83935EcFe4729c4Ea592Ab2bC1A32588409797', // crvOBTC v1
-    '0x123964EbE096A920dae00Fb795FFBfA0c9Ff4675', // crvPBTC v1
-    '0xA8B1Cb4ed612ee179BDeA16CCa6Ba596321AE52D', // crvBBTC v1
-    '0x07FB4756f67bD46B748b16119E802F1f880fb2CC', // crvTBTC v1
-    '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE', // ETH v1
```

Comment:
ETH vault is missing from the `migrationWhitelist` json file

Reply:
It was never in the migrationWhitelist file before and still migrated just fine; I'm not sure if removing it from the retired list will impact the custom APY tooltip and value though